backend/account/tests/test_query.py

```diff
@@ -122,3 +123,11 @@ def test_invalid_remove_friend(self):
             query.remove_friend(user.id, -1)
         with self.assertRaises(ObjectDoesNotExist):
             query.remove_friend(-1, -1)
+
+
+class GetUserByTokenFieldsTests(TestCase):
```

Comment:
Added a basic test that `last_login` is being updated to *something* more recent when `update_last_user_login` is called - hard to test precisely though since the function assigns down to the millisecond (or less not sure exactly)

Reply:
Yeah that's perfect. I wasn't sure if the nature of automated tests would play nice with `timezone`, so I think that's about as precise as we can get.